src/encoded/types/file.py

```diff
@@ -706,6 +706,89 @@ def extra_files_creds(self):
                 extras.append(extra)
             return extras
 
+    def get_presigned_url_location(self, external, request, filename) -> str:
+        """ Opens an S3 boto3 client and returns a presigned url for the requested file to be downloaded"""
+        conn = boto3.client('s3')
+        param_get_object = {
+            'Bucket': external['bucket'],
+            'Key': external['key'],
+            'ResponseContentDisposition': "attachment; filename=" + filename
+        }
+        if request.range:
+            param_get_object.update({'Range': request.headers.get('Range')})
+        location = conn.generate_presigned_url(
+            ClientMethod='get_object',
+            Params=param_get_object,
+            ExpiresIn=36*60*60
```

Comment:
Very small point but if you 
```
from dcicutils.misc_utils import as_seconds
```
you can use `as_seconds(hours=36)` here and it might be a little clearer.